components/schemes/Typography/src/MDCTypographyScheme.h

```diff
@@ -168,6 +168,14 @@ typedef NS_ENUM(NSInteger, MDCTypographySchemeDefaults) {
  */
 @property(nonatomic, assign, readwrite) BOOL useCurrentContentSizeCategoryWhenApplied;
 
+/**
+ If provided and the fonts are scalable, then the font returned by this scheme's properties will be
+ scaled to match the trait environment.
```

Comment:
`mdc_scaledFontForTraitEnvironment:`'s documentation mentions the behavior when device is pre/post iOS 10 and in/ not in an extension.  I am not sure if it is something concerning because the behavior under the hood is not documented here?